src/app/games/conspirateurs/ConspirateursMinimax.ts

```diff
@@ -0,0 +1,111 @@
+                    moves = moves.concat(this.getListJumps(state, coord));
+                }
+            }
+        }
+        return moves;
+    }
+    private getListSimpleMoves(state: ConspirateursState, coord: Coord): ConspirateursMoveSimple[] {
+        const moves: ConspirateursMoveSimple[] = [];
+        const targets: Coord[] = [
+            new Coord(coord.x + 1, coord.y),
+            new Coord(coord.x - 1, coord.y),
+            new Coord(coord.x, coord.y + 1),
+            new Coord(coord.x, coord.y - 1),
+            new Coord(coord.x + 1, coord.y + 1),
+            new Coord(coord.x + 1, coord.y - 1),
+            new Coord(coord.x - 1, coord.y + 1),
+            new Coord(coord.x - 1, coord.y - 1),
+        ];
+        for (const target of targets) {
+            const move: MGPFallible<ConspirateursMoveSimple> = ConspirateursMoveSimple.of(coord, target);
+            if (move.isSuccess() && ConspirateursRules.get().simpleMoveLegality(move.get(), state).isSuccess()) {
+                moves.push(move.get());
+            }
+        }
+        return moves;
+    }
+    private getListJumps(state: ConspirateursState, start: Coord): ConspirateursMoveJump[] {
+        let moves: ConspirateursMoveJump[] = [];
+        for (const firstTarget of ConspirateursRules.get().jumpTargetsFrom(start)) {
+            const jump: ConspirateursMoveJump = ConspirateursMoveJump.of([start, firstTarget]).get();
+            if (ConspirateursRules.get().jumpLegality(jump, state).isSuccess()) {
+                moves.push(jump);
+                moves = moves.concat(this.getListJumpStartingFrom(state, jump));
+            }
+        }
+        return moves;
+    }
+    private getListJumpStartingFrom(state: ConspirateursState, jump: ConspirateursMoveJump): ConspirateursMoveJump[] {
+        const nextJumps: ConspirateursMoveJump[] = ConspirateursRules.get().nextJumps(jump, state);
+        let jumps: ConspirateursMoveJump[] = nextJumps;
+        for (const nextJump of nextJumps) {
+            jumps = jumps.concat(this.getListJumpStartingFrom(state, nextJump));
+        }
+        return jumps;
+    }
+    public getBoardValue(node: ConspirateursNode): NodeUnheritance {
+        const state: ConspirateursState = node.gameState;
+        let score: number = 0;
+        const piecesInShelters: [number, number] = [0, 0];
+        for (let y: number = 0; y < ConspirateursState.HEIGHT; y++) {
+            for (let x: number = 0; x < ConspirateursState.WIDTH; x++) {
+                const coord: Coord = new Coord(x, y);
+                const player: Player = state.getPieceAt(coord);
+                if (player !== Player.NONE) {
+                    const distanceToSide: number = Math.min(Math.min(x, y),
+                                                            Math.min(ConspirateursState.WIDTH - x,
+                                                                     ConspirateursState.HEIGHT - y));
```

Comment:
prefer three declaration to three lines declarations :D constantify Min, als u blief :p !

Reply:
gefait